extractor/extract.py

```diff
@@ -16,4 +20,33 @@ def extract_signature(img: np.ndarray, enable_debug=False) -> np.ndarray:
     img = resize_and_recolor(img, debug=debug)
     img = threshold(img, debug=debug)
     img = autocrop(img, debug=debug)
+    if final_size:
+        img = resize_fixed_aspect(img, final_size=final_size, debug=debug)
+    if bw:
+        img = cv2.adaptiveThreshold(
```

Comment:
I don't think we need an adaptive threshold here -- the image should be pretty much blank and white other than a bit of gray on the edges from the final blur / morphology steps. A simple global threshold like `cv2.threshold(img, 127, 255, cv2.THRESH_BINARY)` should work fine here.